test/new-e2e/tests/agent-platform/persisting-integrations/persisting_integrations_test.go

```diff
@@ -0,0 +1,212 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+package persistingintegrations
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Why is this inside `agent-platform`? Would have assumed it would be in `test/new-e2e/tests/persisting-integrations`

Reply:
We did that because in the beginning we wanted to add a test to the existing `test/new-e2e/tests/agent-platform/upgrade` test. But we couldn't change it, so we shared the codebase and added it next to it.

Reply:
Ok, possible follow up: migrating common code to `pkg` and move up these tests - not urgent